openjdk/excludes/vendors/azul/ProblemList_openjdk8.txt

```diff
@@ -0,0 +1,131 @@
+############################################################################
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+
+# hotspot_all
+
+############################################################################
+
+# hotspot_jre
+runtime/containers/docker/TestMemoryAwareness.java https://bugs.openjdk.java.net/browse/JDK-8229182 generic-all
```

Comment:
The status of https://bugs.openjdk.org/browse/JDK-8229182 is fixed. And it only happened to specific platforms for temurin. That's why I didn't move it to the shared problemlist. Should we just move it to the shared problemlist in case it may happen to other platforms?

Reply:
then let's just leave as is and merge it as it is for now 👍